### PR TITLE
Oz merge

### PR DESCRIPTION
**Pull request by ZumZoom**

### CODE REVIEW

File: contracts/helpers/ChainlinkCalculator.sol [hunk 1]
```diff
@@ -15,33 +15,38 @@ contract ChainlinkCalculator {
...
+        (, int256 latestAnswer2,, uint256 latestTimestamp2,) = oracle2.latestRoundData();
         // solhint-disable-next-line not-rely-on-time
-        require(oracle1.latestTimestamp() + _ORACLE_EXPIRATION_TIME > block.timestamp, "CC: stale data O1");
+        require(latestTimestamp1 + _ORACLE_EXPIRATION_TIME > block.timestamp, "CC: stale data O1");
         // solhint-disable-next-line not-rely-on-time
-        require(oracle2.latestTimestamp() + _ORACLE_EXPIRATION_TIME > block.timestamp, "CC: stale data O2");
+        require(latestTimestamp2 + _ORACLE_EXPIRATION_TIME > block.timestamp, "CC: stale data O2");
 
-        return amount * spread * oracle1.latestAnswer().toUint256() / oracle2.latestAnswer().toUint256() / _SPREAD_DENOMINATOR;
+        return amount * spread * latestAnswer1.toUint256() / latestAnswer2.toUint256() / _SPREAD_DENOMINATOR;
```
Comment (k06a) [inline contracts/helpers/ChainlinkCalculator.sol:50]: This line expects oracle decimals to match. Is is true for all the Chainlink oracles?

File: contracts/OrderRFQMixin.sol [hunk 2]
```diff
@@ -56,7 +56,7 @@ abstract contract OrderRFQMixin is EIP712, AmountCalculator, Permitable {
         bytes calldata signature,
         uint256 makingAmount,
         uint256 takingAmount
-    ) external returns(uint256 /* actualMakingAmount */, uint256 /* actualTakingAmount */) {
+    ) external returns(uint256, uint256) {
```
Comment (Pzixel) [inline contracts/OrderRFQMixin.sol:59]: I know this was reported but if params were removed from here it should document what's first and what's second argument. It's not clear now when it's just a tuple

File: contracts/helpers/ChainlinkCalculator.sol [hunk 3]
```diff
@@ -15,33 +15,38 @@ contract ChainlinkCalculator {
     uint256 private constant _ORACLE_EXPIRATION_TIME = 30 minutes;
     uint256 private constant _INVERSE_MASK = 1 << 255;
 
-    /// @notice Calculates price of token relative to ETH scaled by 1e18
+    /// @notice Calculates price of token realtive to oracle unit (ETH or USD)
```
Comment (Pzixel) [inline contracts/helpers/ChainlinkCalculator.sol:18]: realtive

File: contracts/helpers/ChainlinkCalculator.sol (hunk 1 above)
Comment (Pzixel) [reply contracts/helpers/ChainlinkCalculator.sol:50]: We have line 41 ensuring decimals match

**Pull request merged**